src/actions/onboardingActions.js

```diff
@@ -222,17 +216,6 @@ export const setupAppServicesAction = (privateKey: ?string) => {
       return;
     }
 
-    // set default in case account cannot be created offline
-    const defaultInitialAssets = transformAssetsToObject(initialAssets);
```

Comment:
I would consider putting native tokens initial assets into store. At least we would have ETH, MATIC, etc always there, wdyt?

Reply:
As it's 0 balance there's no use case in the app as it will be ignored in UI so I don't see a need, wdyt?

Reply:
I was thinking about it as a failover measure if initial get supported token call fails. That we could at least display native tokens for each. Not sure if we want to do it, just tossing a suggestion.